automation/taskcluster/decision_task.py

```diff
@@ -71,10 +71,34 @@ def pr_or_push(is_push):
         BUILDER.craft_ktlint_task,
         BUILDER.craft_lint_task,
         BUILDER.craft_compare_locales_task,
+        BUILDER.craft_ui_tests_task,
     ):
         other_tasks[taskcluster.slugId()] = craft_function()
 
-    if is_push and SHORT_HEAD_BRANCH == 'master':
+    return (build_tasks, signing_tasks, other_tasks)
+
+
+def push():
```

Comment:
Thanks for splitting this into two functions, that's :ok_hand: :)

Reply:
🙏